src/avt/Database/Formats/avtMTMDFileFormat.h

```diff
@@ -119,6 +119,7 @@ class DATABASE_API avtMTMDFileFormat : public avtFileFormat
 
     virtual void           ActivateTimestep(int ts)
                                { avtFileFormat::ActivateTimestep(); };
+    using avtFileFormat::ActivateTimestep;
```

Comment:
why is this necessary?

Reply:
There are 2 overloads of `ActivateTimestamp`. Without this, the one overload here hides the second one from the base class. This `using` statement brings overloads from the baseclass that aren't overridden here again.

See [this SO question](https://stackoverflow.com/questions/18515183/c-overloaded-virtual-function-warning-by-clang).